structure/extension/_END.yaml

```diff
@@ -0,0 +1,35 @@
+%YAML 1.2
+---
+lang: en-US
+
+type: structure
+
+uri: https://github.com/glamberson/gedcom-extended-ASSO/_END
+
+extension tags: [ _END ]
+
+specification:
+  - Association End Date
+  - |
+    The date when an association ended. Uses standard GEDCOM 7 date format,
+    allowing for exact dates, date ranges, and approximate dates.
```

Comment:
Is a DatePeriod disallowed? Assuming so, say so.
It wouldn't seem to have any meaning here.